zarr/codecs.py

```diff
@@ -485,6 +485,105 @@ def __repr__(self):
 codec_registry[Delta.codec_id] = Delta
 
 
+class AsType(Codec):
+    """Filter to convert data between different types.
+
+    Parameters
+    ----------
+    dtype : dtype
+        Data type used to store the data.
+    astype : dtype, optional
+        Data type of the data read in.
+
+    Notes
+    -----
+    If `astype` is an integer data type, please ensure that it is
```

Comment:
Sure, this was copy-pasted. I think we should have some note, but not the one here currently.